docs/File-Formats.md

```diff
@@ -1298,15 +1298,15 @@ The cells contain the p-value for the GSVA score: A real number, between 0.0 and
 YAML or JSON file which contains extra information about the cancer study. No compulsory fields are required for this file (free-form). To enable this feature, you need to add a line in the cancer study meta file with `tags_file:` followed the YAML/JSON file name. The information on the YAML or JSON file will be displayed in a table when mousing over a tag logo in the studies on the query page.
 
 ## Generic Assay
-Generic Assay is a two dimensional matrix generalized to capture non-genetic measurements per sample. Instead of a gene per row and a sample per column, a Generic Assay file contains a generic entity per row and a sample per column. A generic entity is defined by the data curator and generally means something other than a gene. Some examples include, treatment response or mutational signatures. For each generic entity - sample pair, a real number represents a captured measurement.
+Generic Assay is a two dimensional matrix generalized to capture non-genetic measurements per sample. Instead of a gene per row and a sample per column, a Generic Assay file contains a generic entity per row and a sample per column. A generic entity is defined by the data curator and generally means something other than a gene. Some examples include, treatment response or mutational signatures. For each generic entity - sample pair, a (real number / text / binary value) represents a captured measurement.
 
 ### Generic Assay meta file
 The generic assay metadata file should contain the following fields:
 ```
 cancer_study_identifier: Same value as specified in meta file of the study
 genetic_alteration_type: GENERIC_ASSAY
 generic_assay_type: <GENERIC_ASSAY_TYPE>, e.g., "TREATMENT_RESPONSE" or "MUTATIONAL_SIGNATURE"
-datatype: LIMIT-VALUE
+datatype: value from LIMIT-VALUE / CATEGORICAL / BINARY
```

Comment:
Just to be consistent with how we define datatype in other profiles, can we update the line to `datatype: LIMIT-VALUE, CATEGORICAL or BINARY`?

Reply:
Sure, updated!

Reply:
Thanks!

Reply:
The failed tests seems to be not related to the changes in this PR. Can we merge?

Reply:
Yes, I think we can merge!